s4e-web/cypress/integration/auth/auth-login.spec.ts

```diff
@@ -28,4 +29,26 @@ context('Auth login', () => {
       .loginAs(this.superAdmin)
       .logout();
   });
+
+  it("shouldn't send empty form", function () {
```

Comment:
You can add all validating form test cases to new context 

 ```
context('valid form', () => {
....
})

```

`should login as zkMember` should be placed after the context

Reply:
Decided to create 2 contexts, and remain beforeEach function for "Valid form" because the previous state would have to be removed from the form.